src/frontend/package.json

```diff
@@ -1,43 +1,31 @@
 {
-	"name": "studio-management-system",
-	"version": "0.1.0",
-	"private": true,
-	"dependencies": {
-		"@testing-library/jest-dom": "^5.16.5",
-		"@testing-library/react": "^13.4.0",
-		"@testing-library/user-event": "^13.5.0",
-		"@types/jest": "^27.5.2",
```

Comment:
ahhh yes it does - fair game erasing it from the repo 💣 - easy to add back later if we feel it






I never write tests for these projects